content/blog/2021-11-10-agent.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Introducing Prometheus Agent Mode, an Efficient and Cloud-Native Way for Metric Forwarding
+created_at: 2021-11-10
+kind: article
+author_name: Bartlomiej Plotka (@bwplotka)
+---
+
+> Bartek Płotka has been a Prometheus Maintainer since 2019 and Principal Software Engineer at Red Hat. Co-author of the CNCF Thanos project. CNCF Ambassador and tech lead for the CNCF TAG Observability. In his free time, he writes a book titled "Efficient Go" with O'Reilly. Opinions are my own!
+
+What I personally love in the Prometheus project, and one of the many reasons why I joined the team, was the laser focus on the project goals. Prometheus was always about moving boundaries when it comes to providing pragmatic, reliable, cheap, yet invaluable metric monitoring. Prometheus ultra-stable and robust APIs, querying language and integration protocols (e.g. Remote Write and [Open Metrics](https://openmetrics.io/)) allowed the Cloud Native Computing Foundation (CNCF) metric ecosystem to grow on those strong foundations. Amazing things happened as a result:
+
+* We can see community exporters for getting metrics about literally everything e.g. [containers](https://github.com/google/cadvisor), [eBPF](https://github.com/cloudflare/ebpf_exporter), [Minecraft server statistics](https://github.com/sladkoff/minecraft-prometheus-exporter) and even [plants health when gardening](https://megamorf.gitlab.io/2019/07/14/monitoring-plant-health-with-prometheus/).
+* Literally, every cloud-native software is expected to have an HTTP `/metric` endpoint that Prometheus can scrape.
```

Comment:
```suggestion
* Literally, every cloud-native software is expected to have an HTTP `/metrics` endpoint that Prometheus can scrape.
```

This statement may be a bit too strong. Literally every? Expected by whom? I would suggest going more for something like "Most people nowadays expect cloud-native software to have an [...]".